Packs/Expanse/Playbooks/Expanse_Incident_Playbook_CHANGELOG.md

```diff
@@ -1,9 +0,0 @@
-## [Unreleased]
```

Comment:
@andrew-expanse Why was this file deleted? Was the 'Expanse Incident Playbook' removed?

Reply:
This was in response to an earlier review comment - https://github.com/demisto/content/pull/7654#issuecomment-648777603

I can add this back if Changelogs for playbooks should not be removed.

Reply:
Oh you are right, you can leave it like this 👌